quinn-proto/src/packet.rs

```diff
@@ -10,16 +10,16 @@ use crate::{
     ConnectionId,
 };
 
-// Due to packet number encryption, it is impossible to fully decode a header
-// (which includes a variable-length packet number) without crypto context.
-// The crypto context (represented by the `Crypto` type in Quinn) is usually
-// part of the `Connection`, or can be derived from the destination CID for
-// Initial packets.
-//
-// To cope with this, we decode the invariant header (which should be stable
-// across QUIC versions), which gives us the destination CID and allows us
-// to inspect the version and packet type (which depends on the version).
-// This information allows us to fully decode and decrypt the packet.
+/// Due to packet number encryption, it is impossible to fully decode a header
```

Comment:
If we're going to make this into a doc string, it should start with a concise description that fits on a single line, separated from the detailed rest of the text by an empty line.